opm/simulators/linalg/findOverlapRowsAndColumns.hpp

```diff
@@ -112,6 +112,28 @@ namespace detail
             }
         }
     }
+    template <class Grid>
+    size_t numInteriorCells(const Grid& grid, bool ownerFirst)
```

Comment:
It is not always the number of interiorCells. At least there should be a comment about this.

Reply:
name changed to numMatrixRowsToUseInSolver.